frontend/tests/controllers/SchoolRankTest.php

```diff
@@ -224,7 +224,9 @@ public function testSchoolGetUsers() {
         }
 
         // User3 automatically organizes a contest
-        $contestData = \OmegaUp\Test\Factories\Contest::createContest();
+        $contestData = \OmegaUp\Test\Factories\Contest::createContest(new \OmegaUp\Test\Factories\ContestParams(
```

Comment:
```suggestion
        $contestData = \OmegaUp\Test\Factories\Contest::createContest(
            new \OmegaUp\Test\Factories\ContestParams(
```
Para reducir la longitud de las líneas de código podemos mandar eso al siguiente renglón